File: IQOUTSOURCING.Domain/DTOs/Auditoria/AuditoriaDTO.cs
using IQOUTSOURCING.Domain.DTOs.Base;
using IQOUTSOURCING.Domain.DTOs.Usuario;

namespace IQOUTSOURCING.Domain.DTOs.Auditoria
{
    public class AuditoriaDTO : BaseDTO
    {
        public DateTime FechaInicioSesion { get; set; }
        public UsuarioDTO? Usuario { get; set; }

    }
}

File: client-app/src/app/modelos/AuditoriaDTO.ts
import UsuarioDTO from "./Usuario/UsuarioDTO";
import UsuarioWithoutPasswordDTO from "./Usuario/UsuarioWithoutPasswordDTO";

export default class AuditoriaDTO{
  id:string = "";
  fechaInicioSesion: string = "";
  usuario: UsuarioWithoutPasswordDTO = {
    id: "",
    userName: "",
    correo: "",
    nombre: ""
  };
}
